ts/Series/Networkgraph/NetworkgraphSeries.ts

```diff
@@ -297,9 +324,41 @@ class NetworkgraphSeries extends Series {
         chart: NetworkgraphChart,
         options: Partial<NetworkgraphSeriesOptions>
     ): NetworkgraphSeries {
-
         super.init(chart, options);
 
+        const dlOptions = this.options.dataLabels;
+
+        // time from init() to the time when dataLabels show up #14398
+        // default set to 2.5 seconds
+        let deferTime = 2500;
+
+        // PROBLEM: if this time is too high
+        // then the dataLabels will never draw because the drawDataLabels()
+        // will not be called after the end of simulation
+        // but how to take care of this edge case?
+
+        // if dataLabels.animation.defer set by the user, use this value
+        if (dlOptions &&
+            dlOptions.animation &&
+            typeof dlOptions.animation !== 'boolean' &&
+            dlOptions.animation.defer
+        ) {
+            deferTime = dlOptions.animation.defer;
+        }
+
+        // @todo: fix casting here
+        // even if the defer is set by the user,
+        // networkgraph will use a custom defer method (with setTimeout)
+        // so we need to set it automatically to a very small number
+        ((dlOptions as any).animation as any).defer = 150;
```

Comment:
In the `Series.prototype.drawDataLabels()` there's a function `getDeferredAnimation` which sets the `animation = { defer, duration }`. If the defer is not set, the fade-in animation will not play. 

If the defer is set to a small value like `150` then the `animationConfig = { defer: 0, duration: 150 }`. Then, we achieve a desired result (_fade-in_ animation which takes 150 miliseconds with no defer time [defer is the `setTimeout`). 

I know hard-coding values like that is not good so I'll look into a better solution for this issue.